app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -279,79 +284,134 @@ class RevisionCardFragmentTest {
     }
   }
 
   @Test
-  fun testRevisionCardTestActivity_configurationChange_toolbarTitle_fractionSubtopicId1_isDisplayedCorrectly() { // ktlint-disable max-line-length
+  fun testRevisionCardTestActivity_fractionSubtopicId1_checkPreviousAndNextNavCardsDisplayed() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID
+        SUBTOPIC_TOPIC_ID_1
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
 
-      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.previous_navigation_card))
+        .check(matches(isDisplayed()))
+
+      onView(withId(R.id.next_navigation_card))
+        .check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testRevisionCardTestActivity_fractionSubtopicId3_checkNextNavCardIsNotDisplayed() {
+    launch<RevisionCardActivity>(
+      createRevisionCardActivityIntent(
+        context,
+        profileId.internalId,
+        FRACTIONS_TOPIC_ID,
+        SUBTOPIC_TOPIC_ID_3
+      )
+    ).use {
       testCoroutineDispatchers.runCurrent()
 
-      onView(withId(R.id.revision_card_toolbar_title))
-        .check(matches(withText("What is a Fraction?")))
+      onView(withId(R.id.next_navigation_card))
+        .check(matches(not(isDisplayed())))
     }
   }
 
   @Test
-  fun testRevisionCardTestActivity_configurationChange_fractionSubtopicId2_checkExplanationAreDisplayedSuccessfully() { // ktlint-disable max-line-length
+  fun testRevisionCardTestActivity_fracSubtopicId1_clickPrevNavCard_opensRevisionCardActivity() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID_2
+        SUBTOPIC_TOPIC_ID_1
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+
+      onView(withId(R.id.previous_navigation_card)).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      intended(hasComponent(RevisionCardActivity::class.java.name))
+      intended(hasExtra(SUBTOPIC_ID_EXTRA_KEY, SUBTOPIC_TOPIC_ID_1 - 1))
+    }
+  }
+
+  @Test
+  fun testRevisionCardTestActivity_fracSubtopicId1_clickNextNavCard_opensRevisionCardActivity() {
+    launch<RevisionCardActivity>(
+      createRevisionCardActivityIntent(
+        context,
+        profileId.internalId,
+        FRACTIONS_TOPIC_ID,
+        SUBTOPIC_TOPIC_ID_1
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+
+      onView(withId(R.id.next_navigation_card)).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      intended(hasComponent(RevisionCardActivity::class.java.name))
+      intended(hasExtra(SUBTOPIC_ID_EXTRA_KEY, SUBTOPIC_TOPIC_ID_1 + 1))
+    }
+  }
+
+  @Test
+  fun testRevisionCardTestActivity_configurationChange_toolbarTitle_fractionSubtopicId1_isDisplayedCorrectly() { // ktlint-disable max-line-length
```

Comment:
For this & next test: please rename so as to not need to disable the max-line-length ktlint check.

Reply:
these tests were not changed/created during this PR, though a variable name was refactored. I suggest getting this changed on a separate issue as I'm working on a limited time availability.